packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -97,8 +79,8 @@ export class GCTelemetryTracker {
 	// Keeps track of unreferenced events that are logged for a node. This is used to limit the log generation to one
 	// per event per node.
 	private readonly loggedUnreferencedEvents: Set<string> = new Set();
-	// Queue for unreferenced events that should be logged the next time GC runs.
-	private pendingEventsQueue: IUnreferencedEventProps[] = [];
+
+	private readonly referencesSinceLastRun: Set<string> = new Set();
```

Comment:
comment please

Reply:
The reason I didn't add comment because it's pretty clear from the name. Not sure what additional information I should add.